rest-api/dao/participant_dao.py

```diff
@@ -201,7 +280,10 @@ def from_client_json(self, resource_json, id_=None, expected_version=None, clien
         providerLink=json.dumps(resource_json.get('providerLink')),
         clientId=client_id,
         withdrawalStatus=resource_json.get('withdrawalStatus'),
-        suspensionStatus=resource_json.get('suspensionStatus'))
+        suspensionStatus=resource_json.get('suspensionStatus'),
+        organizationID=resource_json.get('organizationID'),
```

Comment:
To be consistent: organizationId, not organizationID (for both).

The bigger issue though, is: the hpoId, organizationId, and siteId fields in the API representation should be strings like "PITT', "WISC_MAD", "hpo-site-bannerphoenix"... and the IDs on the participant and participant summary objects in the model are foreign keys to the underlying database rows. So you need to do the conversion from strings to IDs here, and IDs to strings in to_client_json, using HPODao, OrganizationDao, and SiteDao.

Note that these don't have to be set, so you'll need to handle the None case here.

For an example of where we do the conversion from ID to string on participant summary today:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_summary_dao.py#L282 @